### PR TITLE
fix(android): fixed like a million things for android

### PR DESCRIPTION
**Pull request by sjelin**
- Make appium default to 1.6.0 (Android N didn't work on 1.5.x)
- Make virtual devices default to `google_apis`
- Don't delete old virtual devices on update
- Update documentation

### CODE REVIEW

**Comment by cnishina** [comment]
will look at this soon. a million things is a lot of things.

**Comment by sjelin** [comment]
If you need me you know where to find me

File: lib/cmds/initialize.ts [hunk 1]
```diff
@@ -90,45 +90,71 @@ function setupHardwareAcceleration(sdkPath: string) {
...
-        if (deviceType.toUpperCase() == 'DEFAULT') {
-          deviceType = 'android';
-        }
-        return targets.concat(apiLevels.map((level) => {
-          return 'sys-img-' + architecture + '-' + deviceType + '-' + level;
-        }));
-      }, []));
+// Get a list of all the SDK download targets for a given set of APIs,
+// architectures, and platforms
+function getAndroidSDKTargets(
```
Comment (cnishina) [inline lib/cmds/initialize.ts]: In the `initialize.ts` file, would prefer functions to be written as:
```
getAndroidSDKTargets: (apiLevels: string[], architectures: string[], platforms: string[],
  oldAVDs: string[]): string[] => {
```

over: 
```
function getAndroidSDKTargets(apiLevels: string[], architectures: string[], platforms: string[],
  oldAVDs: string[]): string[] {
```

File: lib/cmds/initialize.ts [hunk 2]
```diff
@@ -90,45 +90,71 @@ function setupHardwareAcceleration(sdkPath: string) {
...
+    apiLevels: string[], architectures: string[], platforms: string[],
+    oldAVDs: string[]): string[] {
+  function getSysImgTarget(architecture: string, platform: string, level: string) {
+    if (platform.toUpperCase() == 'DEFAULT') {
+      platform = 'android';
+    }
+    return 'sys-img-' + architecture + '-' + platform + '-' + level;
+  }
+
+  var targets = apiLevels
```
Comment (cnishina) [inline lib/cmds/initialize.ts]: prefer `let` over `var`

**Comment by cnishina** [comment]
Also, might be helpful to have a better commit message.

**Comment by sjelin** [comment]
The commit message does have more detail, but github is only showing the first line.  Still, I'll replace "like a million" with "four" 😛

**Comment by sjelin** [comment]
All comments addressed

File: lib/cmds/initialize.ts (hunk 2 above)
Comment (sjelin) [inline lib/cmds/initialize.ts]: DONE

Comment (sjelin) [inline lib/cmds/initialize.ts]: (Sorry, caps lock)

**Pull request merged**